stable/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 3.5.0
+version: 3.5.1
```

Comment:
Could you bump minor version instead of patch, according to semver? this is new feature.